qa/integration-tests/src/test/java/io/camunda/zeebe/it/management/ControlledActorClockEndpointIT.java

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.it.management;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.model.bpmn.Bpmn;
+import io.camunda.zeebe.protocol.jackson.record.AbstractRecord;
+import io.camunda.zeebe.test.util.actuator.ClockActuatorClient;
+import io.camunda.zeebe.test.util.testcontainers.ZeebeTestContainerDefaults;
+import io.zeebe.containers.ZeebeContainer;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpRequest.BodyPublishers;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Collectors;
+import org.agrona.CloseHelper;
+import org.elasticsearch.client.RestClient;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.Network;
+import org.testcontainers.elasticsearch.ElasticsearchContainer;
+import org.testcontainers.shaded.org.awaitility.Awaitility;
+import org.testcontainers.utility.DockerImageName;
+
+public class ControlledActorClockEndpointIT {
+  private static final String ELASTICSEARCH_HOST = "elasticsearch";
+  private static final String INDEX_PREFIX = "exporter-clock-test";
+  private Network network;
+  private ElasticsearchContainer elasticsearchContainer;
+  private ZeebeContainer zeebeContainer;
+  private ZeebeClient zeebeClient;
+  private final HttpClient httpClient = HttpClient.newHttpClient();
+  private final ObjectMapper mapper = new ObjectMapper();
+
+  private ClockActuatorClient clockActuatorClient;
+
+  @BeforeEach
+  void startContainers() {
+    network = Network.newNetwork();
+    startElasticsearch();
```

Comment:
:upside_down_face: Maybe we can even start the containers in parallel? :thinking: e.g. `Stream.of(elasticsearchContainer, zeebeContainer).parallel().forEach(Startable::start);` or some such. Zeebe should be able to cope with Elastic not being there.

Reply:
Similarly I guess we can close them in parallel, but in this case we probably don't gain that much time anyway.